cql3/query_processor.cc

```diff
@@ -658,7 +665,18 @@ query_processor::parse_statements(std::string_view queries) {
 }
 
 service::migration_manager& query_processor::get_migration_manager() noexcept {
-    return _mm;
+    assert(_remote);
```

Comment:
no assert please, use `on_internal_error()` instead.

Reply:
In this commit `_remote` is always engaged through construction; in the next commit I change the `assert` to `on_internal_error`

(but I see I forgot to do it for `raft_group0_client` - will fix)